spec/models/term_spec.rb

```diff
@@ -138,4 +138,68 @@
       end
     end
   end
+
+  describe "#values_for_property" do
+    context "When requesting a list of values for a property" do
+      let(:label) {RDF::Literal("blah", :language => :en)}
+      let(:resource) do
+        t = Term.new("1")
+        t.label = [label]
+        t
+      end
+
+      it "should return the list of values" do
+        expect(resource.values_for_property(:label)).to eq ["blah"]
+      end
+    end
+
+    context "When requesting a list of values with multiple values" do
```

Comment:
This context and tests looks to be the exact same as the context starting at line #185